components/org.wso2.carbon.identity.oauth.dcr.endpoint/src/main/resources/api.identity.oauth.dcr.endpoint.yaml

```diff
@@ -107,7 +107,7 @@ paths:
         "redirect_uris":["https://client.example.org/callback"],
         "client_name":"application_owner_application_1",
         "grant_types": ["password"]}
-      operationId: getApplicationByName
+      operationId: getApplicationWithName
```

Comment:
what is the reason for this change?

Reply:
Previously this method was named as `getApplicationWithName` in RegisterApi class. But yaml file had the name `getApplicationByName` so auto generated RegisterApi class had that function as `getApplicationByName`. So i changed the yaml file to get the original method name `getApplicationWithName`.